packages/components/src/templates/next/layouts/Content/Content.tsx

```diff
@@ -136,7 +140,14 @@ const createContentLayoutStyles = tv({
     container:
       "mx-auto grid max-w-screen-xl grid-cols-12 px-6 py-12 md:px-10 md:py-16 lg:gap-6 xl:gap-10",
     siderailContainer: "relative col-span-3 hidden lg:block",
-    content: "col-span-12 flex flex-col gap-16 lg:col-span-9 lg:ml-24",
+    content: "col-span-12 flex flex-col gap-16 lg:col-span-9",
+  },
+  variants: {
+    isSideRailPresent: {
+      true: {
```

Comment:
hmmm might we want to ensure siderail is hidden too since now we have this variant prop

```suggestion
      true: {
        siderailContainer: "lg:block col-span-3"
```

then remove that on the root too haha

Reply:
Got a conditional render below that renders the siderail only if it is not null